plugins/chat/spec/models/category_spec.rb

```diff
@@ -22,4 +22,33 @@
       end
     end
   end
+
+  describe "#deletable_for_chat?" do
+    subject(:category) { Fabricate.build(:category) }
+
+    context "when no category channel is present" do
+      it "returns true" do
+        expect(category).to be_deletable_for_chat
+      end
+    end
+
+    context "when a category channel is present" do
+      let(:channel) { Fabricate(:category_channel) }
+      let(:category) { channel.chatable }
+
+      context "when it has chat messages" do
+        let!(:chat_message) { Fabricate(:chat_message, chat_channel: channel) }
```

Comment:
Is there any reason we just don't use directly fabricate inside `it {}` or even `before {}`, it feels more right to me as we don't care about the variable

Reply:
Question of habit I guess :sweat_smile: https://rspec.rubystyle.guide/#let-blocks